agent/tool-scripts/postprocess/unittests

```diff
@@ -145,6 +145,17 @@ function testhaproxy_ocp {
     return $?
 }
 
+function testjmap {
+    echo "Testing jmap under sample $1 ..."
+
+    mkdir /tmp/GenData/$1
+    cp samples/$1/* /tmp/GenData/$1/
+    _TEST_ALTERNATE_TOOLS_LIBRARY=$(pwd) ./jmap-postprocess /tmp/GenData/$1 > /tmp/GenData/$1/stdout 2> /tmp/GenData/$1/stderr
+
+    diff -cr gold/$1/ /tmp/GenData/$1
```

Comment:
We need to add a sample output of the postprocess script to the gold dir in order to compare.

Reply:
done